test/data/testBOMD.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
```

Comment:
It's a new calculation type, so yes.

Reply:
Right, but other time-dependent outputs are in principle possible, so maybe we should call it `testMD`? It feels overly specific to treat BOMD as a calculation type on par with SP and GeoOpt.

Reply:
I suppose it can just be called MD, for the sake of parsing time, but I know that Q-Chem and especially NWChem classical MD output files look very different. If it's purely classical, there won't be `scfenergies`, for example.

Reply:
> I suppose it can just be called MD, for the sake of parsing time, but I know that Q-Chem and especially NWChem classical MD output files look very different. If it's purely classical, there won't be scfenergies, for example.

That's a good point. So leaving `bomd` is fine, since it's probably the simplest and most common dynamics with electronic structure(?).

As far as classical MD is concerned, do we want to parse that as well in the long term?